src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PortableInfoboxObject.java

```diff
@@ -70,12 +74,19 @@ public String getExternalLinkName(int index) {
     return externalLinks.get(index).getText();
   }
 
-  public String getUrlFromExternalLinkaAfterPageIsLoaded() {
+  public String getUrlFromExternalLinkAfterPageIsLoaded() {
     wait.forElementVisible(bodyElement);
 
     return driver.getCurrentUrl();
   }
 
+  public String getHeaderName(int index) {
+    Assertion.assertFalse(headers.isEmpty());
+    wait.forElementVisible(headers.get(index));
+
+    return headers.get(index).getText();
+  }
+
   // TODO: This is not real tap, replace with PerformTouchActions class methods
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)